axelor-supplychain/src/main/java/com/axelor/apps/supplychain/exception/IExceptionMessage.java

```diff
@@ -214,4 +214,7 @@
 
   public static final String SALE_ORDER_STOCK_MOVE_CREATED = /*$$(*/
       "Stock move %s has been created for this sale order" /*)*/;
+
+  static final String STOCK_MOVE_VERIFY_PRODUCT_STOCK_ERROR = /*$$(*/
+      "Product stock for %s is not enough for availability request" /*)*/;
```

Comment:
No needs to put %s and replace it with product names, because if there are many products it will not display properly